src/main/resources/application.properties

```diff
@@ -1,4 +1,4 @@
-logging.file.name=oh-rest-api.log
+# logging.file.name=oh-rest-api.log
```

Comment:
@ArnaudFonzam why this line is commented?

Reply:
i don't no

Reply:
@npasquetto any idea?

anyway I'm moving .properties outside the jar and converting them in .dist (to be copied and configured for each production env)